x-pack/legacy/plugins/apm/public/components/app/ErrorGroupOverview/List/index.tsx

```diff
@@ -59,9 +59,29 @@ const ErrorGroupList: React.FC<Props> = props => {
   const columns = useMemo(
     () => [
       {
-        name: i18n.translate('xpack.apm.errorsTable.groupIdColumnLabel', {
-          defaultMessage: 'Group ID'
-        }),
+        name: (
+          <EuiToolTip
```

Comment:
@mohinderps Thanks for submitting a pull request for this. I noticed that you've implemented it with a regular `EuiIcon`, but we actually have a specific component for something like this; the `EuiIconTip` which is documented here https://elastic.github.io/eui/#/display/tooltip - would you mind refactoring your implementation to use this?

Reply:
Thanks for reviewing it @formgeist . I have done the changes that you suggested.